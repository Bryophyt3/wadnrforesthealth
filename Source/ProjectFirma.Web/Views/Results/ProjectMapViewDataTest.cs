using System.Collections.Generic;
using ApprovalTests;
using ApprovalTests.Reporters;
using ProjectFirma.Web.Views.Shared.ProjectLocationControls;
using ProjectFirma.Web.Models;
using Newtonsoft.Json.Linq;
using NUnit.Framework;

namespace ProjectFirma.Web.Views.Results
{
    [TestFixture]
    public class ProjectMapViewDataTest
    {
        [Test]
        [UseReporter(typeof(DiffReporter))]
        public void EnsureProjectMapCustomizationSignatureTest()
        {
            var projectLocationFilter = new ProjectMapCustomization(ProjectLocationFilterType.FocusArea, new List<int> { 2, 3, 4 }, ProjectColorByType.ProjectStage);
            Approvals.Verify(JObject.FromObject(projectLocationFilter));
        }
    }
}