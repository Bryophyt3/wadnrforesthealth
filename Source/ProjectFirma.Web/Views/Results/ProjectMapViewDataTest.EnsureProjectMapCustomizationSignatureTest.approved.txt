{
  "ProjectLocationFilterType": {
    "ProjectLocationFilterTypeID": 1,
    "ProjectLocationFilterTypeName": "FocusArea",
    "ProjectLocationFilterTypeNameWithIdentifier": "FocusAreaID",
    "ProjectLocationFilterTypeDisplayName": "Focus Area",
    "SortOrder": 10,
    "DisplayGroup": 1,
    "PrimaryKey": 1,
    "ToEnum": 1
  },
  "ProjectColorByType": {
    "ProjectColorByTypeID": 2,
    "ProjectColorByTypeName": "ProjectStage",
    "ProjectColorByTypeNameWithIdentifier": "ProjectStageID",
    "ProjectColorByTypeDisplayName": "Stage",
    "SortOrder": 20,
    "PrimaryKey": 2,
    "ToEnum": 2
  },
  "FilterPropertyValues": [
    2,
    3,
    4
  ],
  "FilterPropertyName": "FocusAreaID",
  "ColorByPropertyName": "ProjectStageID"
}